src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,6 @@
-const numberToReversedDigits = (number) => {};
-
-module.exports = numberToReversedDigits;
+const numberToReversedDigits = (number) => {
+    const reversedArray = Array.from(String(number), n => Number(n));
+    reversedArray.reverse();
+    return reversedArray;
+  };
+  module.exports = numberToReversedDigits;
```

Comment:
You have the same issue with indentation as above here, also you dont need to declare the variables here, you can just chain the methods together and return the result directly:
```
const numberToReversedDigits = (number) =>
  Array.from(String(number), (n) => Number(n)).reverse();

module.exports = numberToReversedDigits;
```